### DISCUSSION TITLE
how can i start contributing to graphback project?

### DISCUSSION THREAD

**Discussion by arnab15** [aerogear/graphback#2253] [Q&A]
I have knowledge of javascript ,nodejs,reactjs,expressjs, Mongodb but I haven't contributed to any open-source project before so I'm a beginner. I would like to contribute to this project.
how should I start can anyone suggest to me the way should I start contributing.

**Comment by craicoverflow** [comment]
Hi @arnab15 and welcome to the Graphback project!

In order to contribute the best thing to do is get familar with Graphback first. Read the website ([graphback.dev](http://graphback.dev/)) and build a TODO application, exercising Graphback's capabilities. Once you have become familiar enough with it this way, contributing in code should be simpler!

Blogging about your experience is a great way to help you learn and is also a good way to begin contributing. Check out this blog one of our most active community members wrote as an example: [How to connect your Graphback project to MongoDB Atlas](https://dev.to/rinkiyakedad/how-to-connect-your-graphback-project-to-mongodb-atlas-5aa8)

**Comment by wtrocki** [comment]
@arnab15 First thing to become a contributor is to understand what project does and become user. Once you use project you bring your own unique experience and take on things - you might see some things to improve, fix or document. From there you can create issue to discuss some of those etc. Graphback is being used in people's templates and as tool for generating GraphQL Apps. 
You can write quick app with it and learn programming

Usual docs like README, CONTRIBUTING files could be helpful here as well.

**Comment by arnab15** [comment]
thank you, sir.
for clearing my all doubts.
First I have to learn about the graphback then I should build a project using it. then I will come back for fixing bugs and contributions.
